projects/RabbitMQ.Client/client/impl/RabbitMQActivitySource.cs

```diff
@@ -0,0 +1,151 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Text;
+using System.Threading;
+using RabbitMQ.Client.Events;
+using RabbitMQ.Client.Framing.Impl;
+
+namespace RabbitMQ.Client
+{
+    public static class RabbitMQActivitySource
+    {
+        // These constants are defined in the OpenTelemetry specification:
+        // https://opentelemetry.io/docs/specs/semconv/messaging/messaging-spans/#messaging-attributes
+        internal const string MessageId = "messaging.message.id";
+        internal const string MessageConversationId = "messaging.message.conversation_id";
+        internal const string MessagingOperation = "messaging.operation";
+        internal const string MessagingSystem = "messaging.system";
+        internal const string MessagingDestination = "messaging.destination.name";
+        internal const string MessagingDestinationRoutingKey = "messaging.rabbitmq.destination.routing_key";
+        internal const string MessagingBodySize = "messaging.message.body.size";
+        internal const string MessagingEnvelopeSize = "messaging.message.envelope.size";
+        internal const string ProtocolName = "network.protocol.name";
+        internal const string ProtocolVersion = "network.protocol.version";
+        internal const string RabbitMQDeliveryTag = "messaging.rabbitmq.delivery.tag";
```

Comment:
Would you expect more attributes in the `messaging.rabbitmq.delivery.` namespace? Otherwise, it'd be better to call it `messaging.rabbitmq.delivery_tag`

Reply:
Will do

Reply:
Done: https://github.com/open-telemetry/semantic-conventions/pull/433